NineChronicles.Headless/GraphTypes/ActionMutation.cs

```diff
@@ -157,6 +164,7 @@ public ActionMutation(NineChroniclesNodeService service)
                             costumes = costumeIds,
                             equipments = equipmentIds,
                             foods = consumableIds,
+                            playCount = playCount < 1 ? 1 : playCount
```

Comment:
In my thought, it'll be better to throw errors when unexpected negative numbers came than forcing to 1 implicitly.

Reply:
I was wondering whether to include that myself.  I adjusted the branch to remove that extra code.